core/modules/image/image.field.inc

```diff
@@ -456,6 +456,12 @@ function image_field_widget_process($element, &$form_state, $form) {
   $element['#theme'] = 'image_widget';
   $element['#attached']['css'][] = backdrop_get_path('module', 'image') . '/css/image.css';
 
+  // Since this is an image field accept only images.
+  $element['upload']['#attributes']['accept'] = 'image/*';
+  // Support for uploads via the camera on mobile devices (.jpg is the default
+  // extension used by cameras in iOS and Android devices).
+  $element['upload']['#attributes']['capture'] = 'camera';
```

Comment:
I've done a little spitballing and if I unset this "capture" attribute completely, iOS will then give the option to select from `Photo Library`, `Take Photo`, or `Browse`. If "capture" is there and blank, or filled with "camera" or "capture" then it goes straight to the camera.

And I've just tested in Firefox on Android and it also seems to give the option to select from `Camera`, or `Files`.

Do we need this "capture" attribute at all?

Reply:
@jenlampton @Retsell  :point_up: